src/Sulu/Bundle/MediaBundle/Media/ImageConverter/ImagineImageConverter.php

```diff
@@ -110,7 +110,7 @@ public function call($command, $parameters)
             $this->image->layers()->coalesce();
             foreach ($this->image->layers() as $layer) {
                 $counter++;
-                $this->commandManager->get($command)->execute($layer, $parameters);
+                $this->commandManager->getCommand($command)->execute($layer, $parameters);
```

Comment:
Isn't `$this->commandManager->getCommand` a bit redundant? I would prefer the old `$this->commandManager->get`, because there shouldn't be any other `get`-method in the future.